labs/lab08-airgap-lite/README.md

```diff
@@ -114,134 +109,132 @@ curl -v https://kubernetes.io
 
 ***
 ## Moving Assets into place
+To perform an Air Gap installation, you will need to collect the following assets:
 
-If you haven't already, you can log out of the Air Gap instance with `exit` or ctrl+D. 
-Our next step is to collect the assets we need for an Air Gap installation:
-
-1. A license with the Air Gap entitlement enabled
-2. An Air Gap bundle containing the kURL cluster components
-3. An Air Gap bundle containing the application components
+* A license with the Air Gap entitlement enabled
+* An Air Gap bundle containing the Kubernetes installer cluster components **
+* An Air Gap bundle containing the application components **
 
-(2) and (3) are separate artifacts to cut down on bundle size during upgrade scenarios where only the application version 
+> ** **Note**: These are separate artifacts to cut down on bundle size during upgrade scenarios where only the application version
 is changing and no changes are needed to the underlying cluster.
 
+If you haven't already, you can log out of the Air Gap instance with `exit` or ctrl+D.
```

Comment:
Maybe we should move this to the previous section? The next step is about getting back on the jumpstation, by using ssh from your workstation. And if they used the multistep aproach to get on the airgap instance, one `exit` will not bring them back onto the workstation.